src/core/src/memory.rs

```diff
@@ -41,55 +41,55 @@ pub enum Usage {
 }
 
 bitflags!(
-    /// Flags providing information about the type of memory access to a resource.
-    ///
-    /// An `Access` value can be a combination of the the following bit patterns:
-    ///
-    /// - [`READ`](constant.READ.html)
-    /// - [`WRITE`](constant.WRITE.html)
-    /// - Or [`RW`](constant.RW.html) which is equivalent to `READ` and `WRITE`.
-    ///
-    /// This information is used to create resources
-    /// (see [gfx::Factory](trait.Factory.html#overview)).
+/// Flags providing information about the type of memory access to a resource.
+///
+/// An `Access` value can be a combination of the the following bit patterns:
+///
+/// - [`READ`](constant.READ.html)
+/// - [`WRITE`](constant.WRITE.html)
+/// - Or [`RW`](constant.RW.html) which is equivalent to `READ` and `WRITE`.
+///
+/// This information is used to create resources
+/// (see [gfx::Factory](trait.Factory.html#overview)).
     #[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
     pub flags Access: u8 {
-        /// Read access
+/// Read access
         const READ  = 0x1,
-        /// Write access
+/// Write access
         const WRITE = 0x2,
-        /// Full access
+/// Full access
         const RW    = 0x3,
     }
 );
 
 bitflags!(
-    /// Flags providing information about the usage of a resource.
-    ///
-    /// A `Bind` value can be a combination of the following bit patterns:
-    ///
-    /// - [`RENDER_TARGET`](constant.RENDER_TARGET.html)
-    /// - [`DEPTH_STENCIL`](constant.DEPTH_STENCIL.html)
-    /// - [`SHADER_RESOURCE`](constant.SHADER_RESOURCE.html)
-    /// - [`UNORDERED_ACCESS`](constant.UNORDERED_ACCESS.html)
-    /// - [`TRANSFER_SRC`](constant.TRANSFER_SRC.html)
-    /// - [`TRANSFER_DST`](constant.TRANSFER_DST.html)
-    ///
-    ///
-    /// This information is used to create resources
-    /// (see [gfx::Factory](trait.Factory.html#overview)).
+/// Flags providing information about the usage of a resource.
```

Comment:
similarly, here and in *all other* bitflag usages:
I think having indentation looks much cleaner, I don't see a point of removing it from the comments